types/webpack-bugsnag-plugins/index.d.ts

```diff
@@ -0,0 +1,151 @@
+// Type definitions for webpack-bugsnag-plugins 1.4
+// Project: https://github.com/bugsnag/webpack-bugsnag-plugins#readme
+// Definitions by: Spencer Miskoviak <https://github.com/skovy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
```

Comment:
It doesn't look like you're using any TS features that require version 3.4.  You might want to specify a lower version to make the types more broadly consumable.

Reply:
Ah, good point 👍 lowered to match webpack's TS version